packages/renderer/src/lib/kube/KubeContextUI.ts

```diff
@@ -44,3 +44,13 @@ export function setKubeUIContextError(contexts: Context[], contextName: string,
 export function clearKubeUIContextErrors(contexts: Context[]): Context[] {
   return contexts.map(ctx => ({ ...ctx, error: undefined }));
 }
+
+export function clearKubeUIContextError(contexts: Context[], contextName: string): Context[] {
+  return contexts.map(ctx => {
+    if (ctx.name === contextName) {
+      return { ...ctx, error: undefined };
+    } else {
+      return ctx;
+    }
+  });
```

Comment:
question: I'm wondering, as we don't really return a copy of the parameters why there is a return of the array.
Either it's always returning a copy of objects or if it's just updating the objects, should be void

Reply:
ok, I changed to always return a clone